react/src/components/FileBrowserModal/FileBrowserModal.tsx

```diff
@@ -45,6 +53,11 @@ const FileBrowserModal = ({
       onCancel={handleClose}
       footer={[
         <Text key="fileCount" type="secondary" style={{ marginRight: 16 }}>
+          {isSingleSelectMode && selectedFiles.length > 1 && (
+            <SectionMessage type="error">
+              Adding multiple assets to a feature is not supported.
```

Comment:
would we want to add this as a string prop multiSelectErrorMessage.

then something like :
```
      {isSingleSelectMode && selectedFiles.length > 1 && (
        <SectionMessage type="error">
          {multiSelectErrorMessage || "Multiple files is not supported"}
        </SectionMessage>
      )}
```
and then

        <FileBrowserModal
          isOpen={isModalOpen}
          toggle={() => setIsModalOpen(false)}
          onImported={handleSubmit}
          allowedFileExtensions={IMPORTABLE_FEATURE_ASSET_TYPES}
          isSingleSelect
          multiSelectErrorMessage="Adding multiple assets to a feature is not supported."
        />